### PR TITLE
Docs/continuous deployment

### PR DESCRIPTION
**Pull request by yock**
@patricksimpson I added some docs on Continuous Deployment with Mina Circle. Could you give it a quick read and let me know what you think?

cc @cromwellryan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,8 +75,31 @@ Change name of the asset and build path.  (gruntfile, gulp, etc)
...
+    # circle.yml
+    deployment:
+      test:
+        branch: master
+        commands:
+          - mina deploy
+
+#### API Permissions
+
+Once this runs, mina-circle will be calling back into CircleCI's API and will require a token. Since it's running on their servers, not your own computer, you'll need to supply one to the project configuration on CircleCI. In your project's settings, find `API permissions` under `Permissions`. Enter a value for `Token label` and click `Create token`, Copy the token string and find the `Environment variables` section under `Tweaks`. That will present you with a form to add a name and value for the variable. The name of the variable should be `MINA_CIRCLE_TOKEN`, and the value is the token you copied in the previous step.
```
Comment (cromwellryan) [inline README.md]: _In your project's settings_ just so there's no confusion, you might try something like _In your CircleCI project settings_...

Comment (cromwellryan) [inline README.md]: Consider turning each statement from _find `API permissions`..._ on into bullets.

File: README.md [hunk 2]
```diff
@@ -73,8 +75,31 @@ Change name of the asset and build path.  (gruntfile, gulp, etc)
...
+
+#### SSH Access
+
+Now running remotely, CircleCI will need a private key which works to access yoru server. [Permissions and access during deployment](https://circleci.com/docs/permissions-and-access-during-deployment) describes setting this up on both CircleCI and your server.
```
Comment (cromwellryan) [inline README.md]: _yoru_ -> _your_

**Comment by cromwellryan** [comment]
Couple formatting items, but otherwise looks good. @nrambeck can we walk through this on kub-templates to verify?

**Comment by patricksimpson** [comment]
I did a read through and I :+1: the changes. Thank you @yock !

**Comment by nrambeck** [comment]
I have a project that isn't setup with Mina Circle yet, so I'll test out the new docs there.

**Comment by nrambeck** [comment]
@yock, you said that it turns out automating deployments may be impossible since artifacts are alway generated last, after the deployment stage. I confirmed this with a test, but did submit an issue ticket to CircleCI to see if they had any way around this.

**Comment by yock** [comment]
@nrambeck Awesome! I'd love to hear what they have to say.

**Pull request merged**